controllers/argocd/configmap.go

```diff
@@ -309,12 +306,6 @@ func newConfigMapWithName(name string, cr *argoproj.ArgoCD) *corev1.ConfigMap {
 	return cm
 }
 
-// newConfigMapWithName creates a new ConfigMap with the given suffix appended to the name.
-// The name for the CongifMap is based on the name of the given ArgCD.
-func newConfigMapWithSuffix(suffix string, cr *argoproj.ArgoCD) *corev1.ConfigMap {
-	return newConfigMapWithName(fmt.Sprintf("%s-%s", cr.ObjectMeta.Name, suffix), cr)
-}
-
```

Comment:
@Rizwana777 did you mean to get rid of this function? it is used for other configmaps as well

Reply:
@jaideepr97 `newConfigMapWithSuffix` is used only used by grafana right now and I didn't see any place in the code where it is used, so I have removed it , I can add it back if we are using it in future somewhere?

Reply:
oops, you're right
my bad